sources/engine/Stride.Assets.Models/AnimationAsset.cs

```diff
@@ -59,6 +59,16 @@ public AnimationAsset()
         [DataMemberIgnore]
         public override UFile MainSource => Source;
 
+        /// <summary>
+        /// Index for FbxAnimStack.
+        /// </summary>
+        /// <userdoc>
+        /// Index of animation to import if multiple are present in the FBX file
```

Comment:
Can you adjust those comments, now that they're not FBX specific?

Reply:
Done! Yeah, I think something like the way material import from fbx is handled is very feasible.

Reply:
@adrsch Did you not push after saying "Done"?

Reply:
Oh... yeah I did forget to push. Oops...